packages/unified-latex-ctan/package/minted/libs/argument-parser.ts

```diff
@@ -16,17 +16,17 @@ const argSpecRDelim: { [delim: string]: Node } = {};
  */
 export const argumentParser: ArgumentParser = (nodes, startPos) => {
     const { argument: optionalArg, nodesRemoved: optionalArgNodesRemoved } =
-        gobbleSingleArgument(nodes, argSpecO, startPos);
+        gobbleSingleArgument(nodes, argSpecO, startPos) as { argument: Argument, nodesRemoved: number };
 
     const { argument: languageArg, nodesRemoved: languageArgNodesRemoved } =
-        gobbleSingleArgument(nodes, argSpecM, startPos);
+        gobbleSingleArgument(nodes, argSpecM, startPos) as { argument: Argument, nodesRemoved: number };
 
     let codeArg: Argument | null = null;
     let codeArgNodesRemoved: number = 0;
     const nextNode = nodes[startPos];
     if (match.group(nextNode)) {
         const mandatoryArg = gobbleSingleArgument(nodes, argSpecM, startPos);
-        codeArg = mandatoryArg.argument;
+        codeArg = mandatoryArg.argument as Argument;
```

Comment:
Prefer testing with `Array.isArray` and throwing an error over casting the type.

Reply:
Fixed at [201461f](https://github.com/siefkenj/unified-latex/pull/42/commits/201461fb3a721645878877726f17d44fc9e396e8)